lib/tail-until.js

```diff
@@ -17,9 +17,12 @@ async function tailUntil (filePath, until, timeout = 5000) {
 
   return new B(async (resolve, reject) => {
     let started = proc.start(startDetector);
+
+    /* eslint-disable */
```

Comment:
What is eslint complaining about here? It is usually better to disable it for a particular rule rather than in general.